eversql/README.md

```diff
@@ -0,0 +1,73 @@
+# EverSQL - Your Database, Just Faster™
+
+## Overview
+- Looking to speed up slow SQL queries?
+- Reached 100% CPU on your database, and you are unsure how to fix it?
+- Database storage is out of space and looking for redundant indexes that you can delete?
+**
+
+[EverSQL][1] is the fastest way to automatically speed up your database and optimize SQL queries.
+80,000+ developers, DBAs, and DevOps engineers use EverSQL for automatic SQL tuning and indexing.
+Our customers report their queries are 5X faster on average, just minutes after getting started.
+Save your team 35 weekly hours on average by optimizing your SQL queries online for free.
+
+EverSQL is 100% non-intrusive, and doesn't access any of your databases' sensitive data.
+
+![snapshot][8]
```

Comment:
Inline images are not supported in the Overview section of the readme - to display an image in the media carousel, add the images to the `media` object in the `manifest.json` file.